ubuntu/Dockerfile

```diff
@@ -1,4 +1,12 @@
+FROM golang:latest as builder
```

Comment:
I was not sure about changing default Dockerfile to ubuntu/Dockerfile. 

If you are golang developer then you might hate to download another docker image for golang which is not necessary as golang developer having all necessary compile & run time environment available. 

It makes sense to have different golang image if you do not have golang environment which is case in dockerhub environment.  As we are building docker image in dockerhub's environment we have to use golang env. to build image.

I think old Dockerfile is more developer friendly who suppose to have golang environment and able to execsute make locally.

Reply:
Yes that's definitely benefit.